go-controller/pkg/ovn/pods.go

```diff
@@ -250,12 +251,17 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) error {
 
 		// If the pod already has annotations use the existing static
 		// IP/MAC from the annotation.
+		if len(podCIDRs) == 1 {
+			addresses = fmt.Sprintf("%s %s", podMac, podCIDRs[0].IP)
+		} else {
+			addresses = fmt.Sprintf("%s %s %s", podMac, podCIDRs[0].IP, podCIDRs[1].IP)
```

Comment:
as above, use `util.JoinIPs` rather than assuming the length

Reply:
Done